### PR TITLE
feat: rex-python initialization

### PR DESCRIPTION
**Pull request by fanzhao1986**
Initialize an empty rex-python project, include package strcutures, tests, github action file

### CODE REVIEW

**Comment by fanzhao1986** [comment]
The action will run the tests on windows, linux and macOS for python 3.7 and 3.8, using pytest and tox framework. The deploy section will not work until, we sort out some configurations in github and pypi.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,63 @@
+[metadata]
+name = rex-python
+description = rex: Reactive Market Python API
```
Comment (markmcdowell) [inline setup.cfg]: Reactive exchange

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+long_description = file: README.md
+version = 0.0.1
+url = http://www.reactivemarkets.com
```
Comment (markmcdowell) [inline setup.cfg]: Reactive exchange.com

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+project_urls =
+    Source=https://github.com/reactiveexchange/rex-python
+    Tracker=https://github.com/reactiveexchange/rex-python/issues
+author = Reactive Markets
+maintainer = Fan Zhao
+author-email = development@reactivemarkets.com
+long_description_content_type = text/markdown
+keywords = reactiveMarket, crossfire
```
Comment (markmcdowell) [inline setup.cfg]: Not crossfire, reactive exchange

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+description = rex: Reactive Market Python API
+long_description = file: README.md
+version = 0.0.1
+url = http://www.reactivemarkets.com
+project_urls =
+    Source=https://github.com/reactiveexchange/rex-python
+    Tracker=https://github.com/reactiveexchange/rex-python/issues
+author = Reactive Markets
+maintainer = Fan Zhao
+author-email = development@reactivemarkets.com
```
Comment (markmcdowell) [inline setup.cfg]: @reactiveexchange.com

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+long_description_content_type = text/markdown
+keywords = reactiveMarket, crossfire
+license = Apache
+license_file = LICENSE
+classifiers =
+        # Classifier list: http://pypi.python.org/pypi?%3Aaction=list_classifiers
+        Development Status :: 5 - Production/Stable
+        Intended Audience :: End Users/Desktop
+        License :: Other/Proprietary License
```
Comment (markmcdowell) [inline setup.cfg]: Apache license

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,64 @@
+[metadata]
+name = rex-python
+description = rex: Reactive Exchange Python API
+long_description = file: README.md
+version = 0.0.1
+url = http://www.reactiveexchange.com
```
Comment (markmcdowell) [inline setup.cfg]: Https

**Comment by marayl** [comment]
I suggest that we rename to `rex-py`. It's shorter, punchier and probably more conventional. @markmcdowell thoughts?

File: setup.cfg [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+version = 0.0.1
+url = https://www.reactiveexchange.com
+project_urls =
+    Source=https://github.com/reactiveexchange/rex-python
+    Tracker=https://github.com/reactiveexchange/rex-python/issues
+author = Reactive Exchange
+maintainer = Fan Zhao
+author-email = development@reactiveexchange.com
+long_description_content_type = text/markdown
+keywords = reactiveMarket, crossfire
```
Comment (markmcdowell) [inline setup.cfg]: Both keywords are still incorrect.

File: setup.cfg (hunk 3 above)
Comment (fanzhao1986) [reply setup.cfg]: skip it before. my bad. Done

**Pull request merged**